scan-citrix-netscaler-version.py

```
@@ -405,9 +409,9 @@ def main() -> None:
         target = target.strip()
         try:
             version = scan_netscaler_version(target, client)
-        except httpx.HTTPError as e:
+        except (httpx.HTTPError, ValueError) as e:
             logging.warning(f"Failed to scan {target}: {e}")
-            version = NetScalerVersion(target, None, None, None, str(e))
+            continue
```

Do you know where the `ValueError` exception came from? Maybe it's better to just catch all exceptions here so it will be properly logged in the `error` field and any other scans can still continue.

Could you change the code block to:

```python
        except Exception as e:
            logging.info(f"Failed to scan {target}: {e}")
            
            version = NetScalerVersion(
                target=target,
                tls_names=None,
                rdx_en_stamp=None,
                rdx_en_dt=None,
                version=None,
                error=str(e),
            )
```

This will ensure that the exception is correctly filled in the error column via keyword argument. Previously this was passed by argument position and was off by one causing it to look weird in the output.